app/controllers/admins_controller.rb

```diff
@@ -14,6 +14,7 @@ def approve_supplier_signup
     @supplier = Supplier.find(params[:id])
     @supplier.approved = true
     if @supplier.save
+      SupplierMailer.new_supplier_approved(@supplier).deliver
```

Comment:
You can use `.deliver_later` instead of `.deliver` for better and fast response of your app